src/config/rt.rs

```diff
@@ -44,16 +44,36 @@ impl RtcBuild {
 pub struct RtcWatch {
     /// Runtime config for the build system.
     pub build: Arc<RtcBuild>,
-    /// Additional paths to ignore.
-    pub ignore: Vec<PathBuf>,
+    /// Paths to watch, defaults to the build target parent directory.
+    pub paths: Vec<PathBuf>,
+    /// Paths to ignore.
+    pub ignored_paths: Vec<PathBuf>,
 }
 
 impl RtcWatch {
     pub(super) fn new(build_opts: ConfigOptsBuild, opts: ConfigOptsWatch) -> Result<Self> {
         let build = Arc::new(RtcBuild::new(build_opts)?);
+
+        let paths = {
+            let mut paths = opts.watch.unwrap_or_default();
+
+            if paths.is_empty() {
+                paths.push(
+                    build
+                        .target
+                        .parent()
+                        .ok_or_else(|| anyhow!("couldn't get parent of {:?}", build.target))?
+                        .to_path_buf(),
+                )
+            }
+
+            paths
+        };
```

Comment:
Looks like this will not include the target dir's parent dir if it is not explicitly declared. That's fine. Just adding a note here. After I merge this, I'll update the docs to state that info.